app/megatron/commands/command_actions.py

```diff
@@ -98,8 +98,8 @@ def forward_message(channel: str, msg: dict, from_user: dict = None) -> dict:
     engagement_channel.last_message_sent = datetime.now(timezone.utc)
     engagement_channel.save()
 
-    megatron_msg, _ = MegatronMessage.objects.update_or_create(
-        integration_msg_id=msg["ts"],
+    megatron_msg, _ = MegatronMessage.objects.exclude(integration_msg_id__isnull=True).update_or_create(
+        integration_msg_id=msg.get("ts"),
         megatron_channel=engagement_channel,
         defaults={"customer_msg_id": response["ts"]},
```

Comment:
@gonzalomolbar is the purpose here to override old `customer_msg_id` with new `customer_msg_id`

Reply:
I didn't write that part, just update it. But i think that is related with keeping in the database only one record of each message, and there was times when the same message sent make two "forward" orders (to open the channel for example, that the customer_msg_id is set as Null).
But I'm not 100% percent sure why is not all the messages saved, even that ones, but it was how it was saved before.
Why added is to be able to save the slash messages, which doesn't generate any integration_msg_id (the ts), and is saved as null.

Reply:
@gonzalomolbar gotcha, let's keep it the same then

Reply:
@gonzalomolbar can `msg["ts"]` be null?

Reply:
When the message it's sended through the slash command it does, that's why I writed it with `get`